riptide-spring-boot-autoconfigure/src/main/java/org/zalando/riptide/autoconfigure/OpenTelemetryPluginFactory.java

```diff
@@ -0,0 +1,15 @@
+package org.zalando.riptide.autoconfigure;
+
+import io.opentelemetry.api.trace.Tracer;
+import org.zalando.riptide.Plugin;
+import org.zalando.riptide.opentelemetry.OpenTelemetryPlugin;
+
+final class OpenTelemetryPluginFactory {
+    private OpenTelemetryPluginFactory() {
+
+    }
+
+    public static Plugin create(final Tracer tracer) {
```

Comment:
Just as a heads up, that if we are considering to have the OTel plugin generate traces and metrics, then this method will have to be extended or overloaded to the get `Meter` (the equivalent of the `Tracer`, but for metrics)

Reply:
I'd advice to maintain two separate plugins within the OpenTelemetry module. One for tracing and one for metrics.

You should also consider not to support the metrics part of OTel at all, since micro meter is already an established standard that can serve as a bridge towards OTel.

Reply:
When this OTel [PR with a Micrometer bridge](https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/4919) gets merged we can just document how to use it in order for the client to emit metrics in OTel format.